internal/server/v1beta1/user_test.go

```diff
@@ -31,35 +31,74 @@ func TestGetUserStarredAssets(t *testing.T) {
 		size     = 10
 	)
 	type testCase struct {
+		UserUUID     string
+		UserID       string
 		Description  string
```

Comment:
Keep this as the first field for the test.

Reply:
Sorry did not get this. Do you mean placing `UserID` above `UserUUID`?

Reply:
The `Description` field should be first in the test case.

Reply:
understood, my bad